opencl/extensions/public/cl_ext_private.h

```diff
@@ -143,3 +143,18 @@ using cl_unified_shared_memory_capabilities_intel = cl_bitfield;
 
 /* cl_queue_properties */
 #define CL_QUEUE_SLICE_COUNT_INTEL 0x10021
+
+/****************************************
+ * cl_intel_pci_bus_info extension *
+ ***************************************/
+#define cl_intel_pci_bus_info 1
+
+// New queries for clGetDeviceInfo:
+#define CL_DEVICE_PCI_BUS_INFO_INTEL 0x410F
```

Comment:
This enum is allocated from an Intel block for this extension.  No concerns from my side.  Thanks!